test/multivariate/mvtdist.jl

```diff
@@ -86,4 +86,50 @@ end
     x = rand(X_implicit)
     @test logpdf(X_implicit, x) ≈ logpdf(X_expicit, x)
 end
+
+@testset "MvNormal affine tranformations" begin
+    @testset "moment identities" begin
```

Comment:
Largely taken from here: 

https://github.com/JuliaStats/Distributions.jl/blob/506ec162912a2edff1b78fc06f274a815f3acd76/test/multivariate/mvnormal.jl#L244

Though if a different form was preferred happy to change them.